class ThrowCF4 
{
	public static void main(String[] args)
	{
		System.out.println("In main");
		throw new ArithmeticException();
	}
}